runbotics/runbotics-actions-windows/src/automation/office/excelErrorMessage.ts

```diff
@@ -68,15 +68,23 @@ export default class ExcelErrorMessage {
         `;
     }
 
+    static existenceOrAbsenceOfWorksheet(shouldExist: boolean): string {
```

Comment:
this contains hard to read Or and Of

also this section is for incorrect input and it's good to include this in function name.

i suggest checkWorksheetInput